### PR TITLE
K8SPG-239 Add 1.3.0 images version

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: api-tests/operator_route_test.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func TestOperatorRouteShouldReturnRigthOperatorVersion(t *testing.T) {
 		{"psmdb-operator", "1.10.0"},
 		{"psmdb-operator", "1.11.0"},
 		{"psmdb-operator", "1.12.0"},
-		{"pg-operator", "1.1.0"},
+		{"pg-operator", "1.3.0"},
```
Comment (tplavcic) [inline api-tests/operator_route_test.go:34]: Shouldn't we have here `1.1.0`, `1.2.0` and `1.3.0` like we have for other operators?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: api-tests/operator_route_test.go (hunk 1 above)
Comment (cap1984) [reply api-tests/operator_route_test.go:34]: I guess we should

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: api-tests/operator_route_test.go [hunk 2]
```diff
@@ -135,7 +136,7 @@ func TestOperatorRoutePgShouldReturnNotEmptyResponses(t *testing.T) {
 		product string
 		version string
 	}{
-		{"pg-operator", "1.1.0"},
+		{"pg-operator", "1.3.0"},
```
Comment (tplavcic) [inline api-tests/operator_route_test.go:139]: For other operators we have all versions listed here.

File: api-tests/operator_route_test.go [hunk 3]
```diff
@@ -32,6 +32,7 @@ func TestOperatorRouteShouldReturnRigthOperatorVersion(t *testing.T) {
 		{"psmdb-operator", "1.11.0"},
 		{"psmdb-operator", "1.12.0"},
 		{"pg-operator", "1.1.0"},
+		{"pg-operator", "1.3.0"},
```
Comment (tplavcic) [inline api-tests/operator_route_test.go:35]: Why skip 1.2.0?

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by tplavcic — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**